phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/processor/AttributeMapConfigProcessor.java

```diff
@@ -0,0 +1,54 @@
+package com.github.phantazmnetwork.api.config.processor;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import it.unimi.dsi.fastutil.objects.Object2FloatMap;
+import it.unimi.dsi.fastutil.objects.Object2FloatOpenHashMap;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+public class AttributeMapConfigProcessor {
+
+    public AttributeMapConfigProcessor() {
+        throw new UnsupportedOperationException();
+    }
+
+    private static final ConfigProcessor<Object2FloatMap<String>> PROCESSOR = new ConfigProcessor<>() {
```

Comment:
should i add a way to create generic Map processors to Ethylene? 

example method signature: 

`<K, V, M extends Map<K, V>> ConfigProcessor<M> ConfigProcessor#mapProcessor(IntFunction<M>, ConfigProcessor<? extends K> keyProcessor, ConfigProcessor<? extends V> valueProcessor)`

or 


`<K, V, M extends Map<K, V>> ConfigProcessor<M> ConfigProcessor#mapProcessor(IntFunction<M>, ConfigProcessor<? extends Map.Entry<K, V>> entryProcessor)`

Reply:
will be added in 0.11.1 which i will push out soon :tm: